python/gcp_client/hydrotools/gcp_client/gcp.py

```diff
@@ -145,14 +90,85 @@ def __init__(self, bucket_name='national-water-model', max_processes=None):
         >>> model_data_service = gcp.NWMDataService()
         
         """
+        # Set bucket name
         self._bucket_name = bucket_name
 
+        # Set max processes
         if max_processes:
             self._max_procs = max_processes
         else:
             self._max_procs = cpu_count() - 2
 
-    def get_blob(self, blob_name) -> bytes:
+        # Set default site mapping
+        if default_site_map != None:
+            self._crosswalk = default_site_map
+        else:
+            self._crosswalk = pd.read_csv(
+                _FEATURE_ID_TO_USGS_SITE_MAP_FILE,
+                dtype={"nwm_feature_id": int, "usgs_site_code": str},
+            ).set_index('nwm_feature_id')
+
+        # Set default dataframe cache
+        self._cache = Path('gcp_cache.h5')
+
+    def list_blobs(
+        self,
+        configuration: str,
+        reference_time: str,
+        must_contain: str = 'channel_rt'
+        ) -> list:
+        """List available blobs with provided parameters.
+
+        Parameters
+        ----------
+        configuration : str, required
```

Comment:
Not sure if you meant in general or with reference to the `NWMDataService` only. I added a reference to `NWMDataService.configurations` property to the docstring. To my knowledge, there is no publicly available, easily/quickly interpreted authoritative source of valid model configurations for all reference times. This may be another of the model's esotericisms we can address.

Reply:
Ah my apologies, your addition of `NWMDataService.configurations` was what I was talking about. I agree with you that we may be able to fill the model esoteric gap in the gh-pages docs addressing the operational model configs. We are great at finding more potential work haha.